portal/app/[locale]/tunnel/_components/submitWithTwoWallets.tsx

```diff
@@ -50,8 +58,21 @@ export const SubmitWithTwoWallets = function ({ disabled, text }: Props) {
   }
 
   return (
-    <Button disabled={disabled} type="submit">
-      {text}
-    </Button>
+    <SubmitWhenConnectedToChain
+      chainId={bitcoin.id}
+      submitButton={
+        <>
+          {validationError ? (
+            <Button disabled type="button">
+              {validationError}
+            </Button>
+          ) : (
+            <Button disabled={disabled} type="submit">
+              {text}
+            </Button>
+          )}
+        </>
```

Comment:
This extra Fragment can be removed
```suggestion
          {validationError ? (
            <Button disabled type="button">
              {validationError}
            </Button>
          ) : (
            <Button disabled={disabled} type="submit">
              {text}
            </Button>
          )}
```